CHANGELOG.rst

```diff
@@ -1,7 +1,9 @@
 3.1.1 (unreleased)
 ==================
 
-* Fix encoding errors for unicode warnings in Python 2.
+* Fix encoding errors for unicode warnings in Python 2. (towncrier: 2436.bugfix)
```

Comment:
@RonnyPfannschmidt  I added my entry here for now, we will want to move them to news fragments for #2431. 😁